app/helpers/tag_helper.rb

```diff
@@ -0,0 +1,12 @@
+module TagHelper
+  def tag_colours
+    %w[primary info success olive warning orange danger pink purple violet]
+  end
```

Comment:
this could probably be a constant

```suggestion
    TAG_COLOURS = %w[primary info success olive warning orange danger pink purple violet]
```